README.md

```diff
@@ -134,12 +134,13 @@ gulp.task('pre-push', guppy.src('pre-push', function (files, extra, cb) {
 }));
 ```
 
-#### guppy.stream(*hookName*)
+#### guppy.stream(*hookName, options*)
```

Comment:
I don't think it is needed - `guppy.src`. As far as I understand from the docs, `guppy.src` is used to provide file list to `gulp.src`, so library users can pass options directly to `gulp.src`.